assets/controller.yaml

```diff
@@ -117,16 +117,15 @@ spec:
           args:
             - --csi-address=$(ADDRESS)
             - --default-fstype=ext4
-            # - --feature-gates=Topology=true
+            - --feature-gates=Topology=true
             # - --extra-create-metadata=true
             - --v=${LOG_LEVEL}
             - --leader-election
             - --leader-election-lease-duration=${LEADER_ELECTION_LEASE_DURATION}
             - --leader-election-renew-deadline=${LEADER_ELECTION_RENEW_DEADLINE}
             - --leader-election-retry-period=${LEADER_ELECTION_RETRY_PERIOD}
             - --http-endpoint=localhost:8202
-            # needed only for topology aware setup
-            # - "--strict-topology"
+            - "--strict-topology"
```

Comment:
Why is this uncommented here and then commented back in this [commit](https://github.com/openshift/vmware-vsphere-csi-driver-operator/pull/107/commits/7d4853dcc988b4222869aa81184113df4973d2ca)? 

Also if this should be uncommented it must not be quoted.

Reply:
oh sorry, those are intermediate commits. End result should be obviously be commented.